### ISSUE TITLE
CRITICAL: QR CODE SCANNING NOT WORKING

### ISSUE THREAD

**Issue by smashah** [open-wa/wa-automate-nodejs#2319]

**Comment by smashah** [comment]
@github-actions run

<details>
<summary>⚡ Release!  ⚡</summary>

```js
(async () => {
function exec(cmd) {
  console.log(execSync(cmd).toString());
}
//set the version type
process.env.VERS = "patch"

// Config
const gitUserEmail = "github-actions[bot]@users.noreply.github.com";
const gitUserName = "github-actions[bot]";

exec(`echo "//registry.npmjs.org/:_authToken=$NPM_TOKEN" > .npmrc`);
exec(`git config --global user.email "${gitUserEmail}"`);
exec(`git config --global user.name "${gitUserName}"`);
exec(`npm i -D`);
exec(`npm run release-ci $VERS`);

// types only package
exec('npm config set //registry.npmjs.org/:_authToken $NPM_TOKEN')
exec(`cd ./types-only && npm version $VERS && npm run build && npm publish && cd ..`);
exec(`git commit -a -m 'updated types-only package'`);
exec(`git push --force`);

//comment on the issue
var result = execSync(`npx auto-changelog -o ./tempchangelog.txt --commit-limit false --template ./compact-keepachangelog.hbs --stdout`).toString();

    await postComment(result);

//create changelog image
exec(`npm run release-image`);
exec(`git commit -a -m 'updated release-image'`);
exec(`git push --force`);
  })();
```
</details>

**Comment by smashah** [comment]
# Changelog

## 🚀 Release [4.24.1](https://github.com/open-wa/wa-automate-nodejs/compare/4.24.0...4.24.1) (2021-11-24)

- Release 4.24.1 [`9f07de3`](https://github.com/open-wa/wa-automate-nodejs/commit/9f07de36b9aba940ebc635945a03d5d1ca6ff8d3)
- 🚑  fix qr codes #2319 [`e5d1d32`](https://github.com/open-wa/wa-automate-nodejs/commit/e5d1d32e5d6ae7693e0a8038b663062a29a2598d)
- 🥚  patches update [`58a738c`](https://github.com/open-wa/wa-automate-nodejs/commit/58a738c548265d202e5f406c7c831545043b2370)
- updated types-only package [`b970ca6`](https://github.com/open-wa/wa-automate-nodejs/commit/b970ca6d9137050ea2f1a09e3dd828b1cfd88645)
